system-tests/projects/sourcemaps/cypress-disabled.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  e2e: {
+    supportFile: false,
+    setupNodeEvents (on, config) {
+      require('./cypress-preprocessor.config.js')(on, config)
```

Comment:
@ryanthemanuel Are there not more disabled/enabled examples that we'd want to test against? Is there concern that our sourcemaps logic could work for some preprocessors but not others with how we're calculating it overall? https://docs.cypress.io/api/node-events/preprocessors-api#Source-maps